### DISCUSSION TITLE
Mevo Plus Connector

### DISCUSSION THREAD

**Discussion by gsgorveatt1234** [springbok/MLM2PRO-GSPro-Connector#118] [Ideas]
I just tried out this connector and its working amazing for web cam putting. Only thing is I have one of the original mevo plus and it didn't come with the pc software of fs golf. My question is, I still have a copy of one of the original connectors one of the users created before flightscope jumped on with gspro. I tried it and it still works and it uses the api version of gspro just like the MLM2PRO-GSPro-Connector. Is there anyway of having 2 connectors running at same time or combining this option with the MLM2PRO-GSPro-Connector.  I believe this would help fix delay users experience as well.

**Comment by wonder99** [comment]
Can you point me to the location of that original mevo connector?  We need to dig into its details.

**Reply by gsgorveatt1234** [reply]
I just have a copy a user made like 3 years ago. I can send you a copy if you like. Not sure if i can add it here or not but let me know. I attached a picture of program. 
![mevoplusconnector](https://github.com/user-attachments/assets/fbd4301a-ea34-4e01-8e9d-9cd369941c6b)

**Reply by gsgorveatt1234** [reply]
Here's what it looks like. 
![image](https://github.com/user-attachments/assets/692d775b-e1e4-4df2-8a64-458515e3b475)

**Reply by gsgorveatt1234** [reply]
Ok so from what I'm reading I need a training file in order for my roi's to get proper numbers. With the program i'm using, it also show's my numbers after each shot on one of the tabs. I was thinking of using this instead of flightscope pc app as it also has force chip option. Is anyone able to help me create trained data file for this image below. Thanks
![mevo plus](https://github.com/user-attachments/assets/2d2cfc36-ccc3-420a-a12a-5a2124fde33d)

**Comment by gsgorveatt1234** [comment]
Just looking at your connector an curious about the relay server. Can I combine the connector above with your connector using the relay server. If so how would I go about doing this. Thanks
